XRTK-Core/Packages/com.xrtk.core/Definitions/Controllers/Simulation/Hands/SimulatedHandControllerPose.cs

```diff
@@ -185,7 +185,7 @@ public void ComputeJointPoses(Handedness handedness, Quaternion rotation, Vector
         public void ParseFromJointPoses(MixedRealityPose[] joints, Handedness handedness, Quaternion rotation, Vector3 position)
         {
             var invRotation = Quaternion.Inverse(rotation);
-            var invCameraRotation = Quaternion.Inverse(MixedRealityToolkit.CameraSystem.CameraRig.PlayerCamera.transform.rotation);
+            var invCameraRotation = Quaternion.Inverse(MixedRealityToolkit.CameraSystem.MainCameraRig.PlayerCamera.transform.rotation);
```

Comment:
We should check if the camera system is null here and fall back to `CameraCache.Main`